tests/BuildScriptGenerator.Tests/Python/PythonPlatformDetectorTest.cs

```diff
@@ -119,11 +124,14 @@ public void Detect_ReutrnsNull_WhenRuntimeTextFileExists_ButDoesNotTextInExpecte
             Assert.Null(result);
         }
 
-        [Fact]
-        public void Detect_ReturnsVersionFromRuntimeTextFile_IfOptionsValueIsNotPresent()
+        [Theory]
+        [InlineData("3")]
+        [InlineData("3.7")]
+        [InlineData("3.7.5")]
+        [InlineData("3.7.5b01")]
```

Comment:
do we need tests for both 3.7.5 and 3.7.5b01?

Reply:
This is just to test that our parsing of runtime text file works for all valid python versions..for example runtime.txt version format is like `python-3.5.5`